### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by thespad**
cstate: https://github.com/linuxserver/cstate/pull/167

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/init-flexget-config/run [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    rm -rf "/config/.flexget/.config-lock"
+fi
+
+if [[ ! -f "/config/.flexget/config.yml" ]]; then
+    cp /default/config.yml /config/.flexget/config.yml
+fi
+
+if [[ -n "${FG_WEBUI_PASSWORD}" ]]; then
+    if ! flexget --loglevel error web passwd "${FG_WEBUI_PASSWORD}" | tee /dev/stderr | grep -q 'Updated password'; then
+        exit 1
```
Comment (nemchik) [inline root/etc/s6-overlay/s6-rc.d/init-flexget-config/run]: should this exit 1? or should it sleep infinity?

Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/init-flexget-config/run]: Yes, but also the FG_WEBUI_PASSWORD should be listed as optional (as you can do it via the CLI and the webui is optional anyway).

Comment (nemchik) [reply root/etc/s6-overlay/s6-rc.d/init-flexget-config/run]: So if the if statement here fails we're exiting the script and not doing the permission ownership at the bottom?

Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/init-flexget-config/run]: Sorry, that was ambiguous. Yes it should have been a sleep. Exiting 1 would halt the init too, but would also cause a loop if the container is set to restart.

**Review by nemchik — APPROVED** [review]

**Pull request merged**